static/js/lib/learning_resources.js

```diff
@@ -26,15 +29,24 @@ export const availabilityLabel = (availability: ?string) => {
 }
 
 export const resourceLabel = (resource: string) => {
-  return concat(capitalize(resource), "s")
+  switch (resource) {
+  case LR_TYPE_USERLIST:
+    return "Learning Paths"
+  default:
+    return concat(capitalize(resource), "s")
+  }
```

Comment:
nit: might be a little more concise to do

```js
export const resourceLabel = (resource: string) =>
  resource === LR_TYPE_USERLIST ? "Learning Paths" : concat(capitalize(resource), "s")
```

Reply:
👍